### PR TITLE
Level 1 TSC Completed

### PR DESCRIPTION
**Pull request by serniko97**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export class Challenge
+{
+    public buzz(n: number): Promise<string|number>
+    {
+        let output: string = "";
+
+        return new Promise<string|number>((resolve, reject) =>
+        {
+            if(n%3 == 0)
```
Comment (Pigotz) [inline src/index.ts:9]: Space around the operator

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                output += "fizz";
+            if(n%5 == 0)
```
Comment (Pigotz) [inline src/index.ts:11]: Space around the operator and space before `if`

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+                output += "buzz";
+            else if(n%3 != 0)
```
Comment (Pigotz) [inline src/index.ts:13]: Space around the operator

**Comment by serniko97** [comment]
It should be okay now

**Pull request closed without merging**